content/projects.md

```diff
@@ -0,0 +1,21 @@
+# Projects
+
+## Organization
+
+TODO: Explain what is the minimal documentation required for any Optakt project.
```

Comment:
Also any public project should have a license (didn't see a license mentioned)

Reply:
That will be part of the minimal documentation there no worries :)